src/utils/dateFormatting.ts

```diff
@@ -1,7 +1,20 @@
-export default function formattingDate(date) {
+export function formattingDate(date: number) {
   return new Date(date).toLocaleString("en-GB", {
     day: "numeric",
     month: "long",
     year: "numeric",
   });
 }
+
+export function formattingHours(date: number) {
```

Comment:
please try to avoid executing the same methods. You can do something like
```
export function formattingHours(pureDate: number) {
  const date = new Date(pureDate);
  const minutes = date.getMinutes();

  return `${date.getHours()}:${minutes < 10 ? "0" : ""}${minutes}`;
}
```